contrib/kube-local/README.md

```diff
@@ -0,0 +1,393 @@
+| CONTAINER_ORCHESTRATION_KUBECONFIG_NAME| Kubeconfig filename. Default: **admin.kubeconfig**
+| CONTAINER_ORCHESTRATION_KUBECONFIG_PATH| Full path for Kubeconfig. Default **/var/run/${CONTAINER_ORCHESTRATION_NAME}/${CONTAINER_ORCHESTRATION_KUBECONFIG_NAME}**
+| CONTAINER_ORCHESTRATION_OUTPUT_DIR| Container Orchestration output dir. Default: **${CONTAINER_ORCHESTRATION_GOLANG_DIR}/_output/bin**
+| CONTAINER_ORCHESTRATION_ETCD| Etcd path. Default **${CONTAINER_ORCHESTRATION_GOLANG_DIR}/third_party/etcd**
+| CONTAINER_ORCHESTRATION_SERVICES| Container Orchestration services. Default: **kube-controller-manager kube-apiserver kubelet kube-scheduler etcd conmon**
+
+### Container Network
+| Name | Description |
+| ------ | ------ |
+| CONTAINER_NETWORK_REPO_NAME| Container Network repo name. Default **containernetworking**
+| CONTAINER_NETWORK_REPO_PLUGIN_DIR| Container Network plugin dir **plugins**
+| CONTAINER_NETWORK_GOLANG_SRC_GITHUB| Container Network Golang src github dir
+| CONTAINER_NETWORK_GOLANG_DIR| Container Network Golang dir. Default: **${GOLANG_PROJECT_HOME_SRC}/${CONTAINER_NETWORK_GOLANG_SRC_GITHUB}/${CONTAINER_NETWORK_REPO_NAME}**
+| CONTAINER_NETWORK_GITHUB_URL| Container Network Github URL. Default: https://github.com/containernetwork/plugins
+| CONTAINER_NETWORK_VERSION| Container Network version. Example: master. Please note: No need to assign if using tag
+| CONTAINER_NETWORK_TAG| Container Network tag. Default: **v0.8.1**
+| CONTAINER_NETWORK_CNI_PATH| Container Network CNI path. Default: **/opt/cni**
+| CONTAINER_NETWORK_CNI_PATH_BIN| Container Network CNI Bin path. Default: **${CONTAINER_NETWORK_CNI_PATH}/bin**
+| CONTAINER_NETWORK_PLUGINS| Container Network plugins: Default: **bandwidth bridge firewall flannel host-device ipvlan loopback macvlan portmap ptp sbr tuning vlan**
+
+### Additional info for the cluster
+| Name | Description |
+| ------ | ------ |
+| IP_LOCALHOST| Local IP address.
+| FEATURE_GATES| Feature gates are a set of key=value pairs that describe Kubernetes features. Default: **AllAlpha=false,RunAsGroup=true**
+| PATH_PROJECTS| Add additional path to ${PATH} env var.
+| CONTAINER_RUNTIME| The container runtime to use. Default: **remote**
+| CGROUP_DRIVER| Cgroup driver. Default: **systemd**
+| CONTAINER_RUNTIME_ENDPOINT| Container runtime endpoint. Default: **${RUNTIME_ENDPOINT}**
+| ALLOW_SECURITY_CONTEXT| Allow Security Context. Default: **","**
+| ALLOW_PRIVILEGED| Allow privileged process. Default: **1**
+| DNS_SERVER_IP| DNS Server IP. Default: **${IP_LOCALHOST}**
+| API_HOST| API Host. Default: **${IP_LOCALHOST}**
+| API_HOST_IP| API Host IP. Default: **${IP_LOCALHOST}**
+| KUBE_ENABLE_CLUSTER_DNS| Default: **true**
+| ENABLE_HOSTPATH_PROVISIONER| Default: **true**
+| KUBE_ENABLE_CLUSTER_DASHBOARD| Default: **true**
+| KUBECONFIG | Path for the Kubeconfig. Default: **${CONTAINER_ORCHESTRATION_KUBECONFIG_PATH}**
+| KUBERNETES_PROVIDER | Default: **local**
```

Comment:
This is really a huge README.md.  For a future PR, it would be good to spin off a couple of pages from this such as the demos and examples and add links from this page to them.

Reply:
sure Tom, thanks for the inputs.